src/work-allocation/containers/my-tasks/my-tasks.component.spec.ts

```diff
@@ -1,19 +1,22 @@
 import { CdkTableModule } from '@angular/cdk/table';
-import {Component, ViewChild} from '@angular/core';
-import {async, ComponentFixture, TestBed} from '@angular/core/testing';
-import {Router} from '@angular/router';
-import {RouterTestingModule} from '@angular/router/testing';
-import {AlertService, LoadingService, PaginationModule} from '@hmcts/ccd-case-ui-toolkit';
-import {ExuiCommonLibModule, FeatureToggleService} from '@hmcts/rpx-xui-common-lib';
-import {of} from 'rxjs';
-import {SessionStorageService} from '../../../app/services';
-
-import {WorkAllocationComponentsModule} from '../../components/work-allocation.components.module';
-import {Task} from '../../models/tasks';
-import {CaseworkerDataService, WorkAllocationTaskService} from '../../services';
-import {getMockTasks} from '../../tests/utils.spec';
-import {TaskListComponent} from '../task-list/task-list.component';
-import {MyTasksComponent} from './my-tasks.component';
+import { Component, ViewChild } from '@angular/core';
+import { async, ComponentFixture, fakeAsync, TestBed, tick } from '@angular/core/testing';
+import { Router} from '@angular/router';
+import { RouterTestingModule } from '@angular/router/testing';
+import { AlertService, LoadingService, PaginationModule } from '@hmcts/ccd-case-ui-toolkit';
+import { ExuiCommonLibModule, FeatureToggleService, FilterService } from '@hmcts/rpx-xui-common-lib';
+import { FilterSetting } from '@hmcts/rpx-xui-common-lib/lib/models/filter.model';
+import { of } from 'rxjs';
+
+import { TaskListComponent } from '..';
+import { SessionStorageService } from '../../../app/services';
+import { AllocateRoleService } from '../../../role-access/services';
+import { WorkAllocationComponentsModule } from '../../components/work-allocation.components.module';
+import { FieldType } from '../../enums';
+import { Task } from '../../models/tasks';
+import { CaseworkerDataService, WASupportedJurisdictionsService, WorkAllocationFeatureService, WorkAllocationTaskService } from '../../services';
+import { getMockTasks } from '../../tests/utils.spec';
+import { MyTasksComponent } from './my-tasks.component';
```

Comment:
Minor comment - incorrect import ordering

Reply:
explained above comment why we should not fine-tune code while doing removing MVP .